specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-03-01/virtualMachineScaleSet.json

```diff
@@ -3947,6 +3947,36 @@
         }
       },
       "description": "The configuration parameters used while performing resilient VM deletion."
+    },
+	 "ScheduledEventsPolicy": {
+      "properties": {
+        "UserInitiatedRedeploy": {
+          "automaticallyApprove": {
+            "type": "boolean",
+            "description": "Specifies Redeply Scheduled Event related configurations."
+          }
+        },
+        "UserInitiatedReboot": {
```

Comment:
Add "type": "object"
and define properties automaticallyApprove and description.